external/js/kyber/src/curve/nist/curve.ts

```diff
@@ -0,0 +1,69 @@
+import { Group, Scalar, Point } from "../../index";
+import NistPoint from "./point";
+import NistScalar from "./scalar"
+import elliptic = require("elliptic");
+import BN = require("bn.js");
+
+export type BNType = string | Buffer | BN
+
+export default class Weierstrass implements Group {
+    curve: any;
+    redN: any;
```

Comment:
how come these have the type `any`? is it because those are objects from an external library? maybe some of those types can be found here https://github.com/DefinitelyTyped/DefinitelyTyped ?

Reply:
If you check the package.json file, you can note that I used this repo for the typing but for a reason I can't explain, they decided not to export the Reduction type... I'm investigating, the pull request was mostly to check the tests but I'm not done yet ;)

Reply:
ah yes, it's still WIP. but it already looks really good!